build.sh

```diff
@@ -82,27 +76,35 @@ prepare_managed_build()
         fi
     fi
 
-    # Grab the MSBuild package if we don't have it already
-    if [ ! -e "$__msbuildpath" ]; then
-        echo "Restoring MSBuild..."
-        mono "$__nugetpath" install $__msbuildpackageid -Version $__msbuildpackageversion -source "https://dotnet.myget.org/F/dotnet-buildtools/" -OutputDirectory "$__packageroot"
-        if [ $? -ne 0 ]; then
-            echo "Failed to restore MSBuild."
-            exit 1
-        fi
-    fi
-    
     # Run Init-Tools to restore BuildTools and ToolRuntime
-    chmod a+x $__scriptpath/init-tools.sh
-    sh $__scriptpath/init-tools.sh
+    $__scriptpath/init-tools.sh
+}
+
+prepare_native_build()
+{
+    # Specify path to be set for CMAKE_INSTALL_PREFIX.
+    # This is where all built CoreClr libraries will copied to.
+    export __CMakeBinDir="$__BinDir"
+
+    # Configure environment if we are doing a clean build.
+    if [ $__CleanBuild == 1 ]; then
+        clean
+    fi
+
+    # Configure environment if we are doing a verbose build
+    if [ $__VerboseBuild == 1 ]; then
+        export VERBOSE=1
+    fi
 }
 
-build_managed_wcf()
+build_managed_corefx()
```

Comment:
I'm afraid I agree we should either consciously diverge or make a PR to CoreFx to generalize their names.  I don't think it is widely appreciated how difficult it is becoming to stay in sync with their changes.